guacamole/src/main/frontend/src/app/auth/service/authenticationService.js

```diff
@@ -18,25 +18,46 @@
  */
 
 /**
- * A service for authenticating a user against the REST API.
+ * A service for authenticating a user against the REST API. Invoking the
+ * authenticate() or login() functions of this service
```

Comment:
It looks like the second half of this sentence is missing?

Reply:
Whoops - looks like I